templates/packages/partner.html

```diff
@@ -0,0 +1,33 @@
+{%extends 'base.html' %}
+
+{% block title%}
+    הזמנות
+{%endblock%}
+{% block content%}
+    <div class="row justify-content-center mb-3">
+        <div class="col text-start">
+            {% include "packages/parts/search.html" %}

```

Comment:
This shows up pretty awkwardly.
![search](https://i.imgur.com/lF1Xg9j.png)

Reply:
Fixed it, please check how layout looks with many cards, as might be the case on many pages. Since I have a very wide screen, even though I tried to optimize of other screen sizes there are still some specific width points (mostly on contrasts of the built in bootstrap sizes) where it doesn't look so good. So just make sure that this points don't fall on a typical laptop width screen.

Reply:
It looks decent, but it is weird how its split to the right side of the screen, whereas the _new order_ is a small button on the left. Maybe put them bellow each other?